source/quick-start.txt

```diff
@@ -7,193 +7,202 @@ Quick Start
+This command downloads ``mongodb`` package and dependencies required for its
+installation and saves them into a directory called ``node_modules`` in
+your project directory, and records the dependency information in the
+``package.json`` file you generated in the previous step.
 
-You can store your application configuration separately for security
-and convenience. We use the `dotenv
-<https://www.npmjs.com/package/dotenv>`_ NPM module to store our
-configuration data in a separate local file called ``.env``. If you are
-using git source control, add the ``.env`` file to your ``.gitignore``
-entries to prevent accidentally sharing contents such as database
-credentials.
+At this point, you should have appropriate versions of Node.js and NPM
+installed as well as a project directory that contains the dependencies you
+need to use the Node.js MongoDB driver.
 
-First, add ``dotenv`` to your project:
+Create a MongoDB Cluster
+------------------------
 
-.. code-block:: sh
+Set up a Free Tier Cluster in Atlas
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-  npm install --save dotenv
+After installing the Node MongoDB driver, create a MongoDB instance to store
+and manage your data. Complete the
+:atlas:`Get Started with Atlas <getting-started>` guide to set up a new
+Atlas account, free tier cluster (MongoDB instance), load datasets, and
+interact with the data.
 
-Next, create the ``.env`` file and add a variable called
-``MONGODB_URI``:
+After completing the steps in the Atlas guide, you should have a new MongoDB
+cluster deployed in Atlas, a new database user, and sample datasets loaded
+into your cluster.
 
-.. code-block:: sh
+Connect to your Cluster
+-----------------------
 
-  touch .env
-  echo "MONGODB_URI=<connection_uri>" >> .env
+In this step, we create and run an application that uses the Node.js MongoDB
+driver to connect to your instance of MongoDB and run a query on the sample
+data.
 
-Add your Atlas Connection String
---------------------------------
+To connect to Atlas using the Node.js driver, we provide instructions to the
+driver on where and how to connect to your instance in a string called the
+*connection string*.
 
-Open your newly created ``.env`` file. If you are unable to locate this
-file in your project directory, you may need to set your system to
-display hidden files. Once your ``.env`` is open, replace
-``<connection_uri>`` with your connection string from Atlas. Ensure you
-update ``<username>`` and ``<password>`` in your connection string.
+Log into your Atlas account and navigate to the **Clusters** section and
+click the **Connect** button for the cluster in which you want to connect
+as shown below.
 
-Connect to your Cluster
------------------------
+.. figure:: /includes/figures/atlas_connection_select_cluster.png
 
-For the final step, we'll write some code that tests your environment's
-connection to your instance of MongoDB.
+Follow the instructions in the dialog to whitelist your IP address and create
```

Comment:
Probably doesn't matter, but if they've completed all steps of the Getting Started with Atlas guide, then they'll have already done these two things (IP whitelist and new MongoDB user).

Reply:
I'll go back and check the Getting Started with Atlas. While I remember there was a step to create a user, I do not remember whether it was through the "Connect" dialog wizard.

Reply:
Ah, I see the user goes through the Python or Mongo Shell example that I thought would be confusing in "Get Started with Atlas". I guess I will update this to only remind them how to navigate to retrieve the info.